static/js/protein/page.tsx

```diff
@@ -44,12 +44,12 @@ import {
   getVarSigAssoc,
   getVarTypeAssoc,
 } from "./data_processing_utils";
-interface PagePropType {
+export interface PagePropType {
```

Comment:
I don't think we should use the same PageProp and PageState for both Protein browser and Disease browser

Reply:
So, do we use the same PageProp and PageState definitions but define them in different files without exporting them?

Reply:
Yes, because we want to be able to update the prop type and state types of each component without having to update the other component